pkgs/by-name/mo/modrinth-app-unwrapped/package.nix

```
@@ -121,6 +121,8 @@ rustPlatform.buildRustPackage (finalAttrs: {
     local nixGradleFlags=()
     concatTo nixGradleFlags gradleFlags gradleFlagsArray
     export NIX_GRADLEFLAGS_COMPILE="''${nixGradleFlags[@]}"
+
+    cp packages/app-lib/.env.local packages/app-lib/.env
```

```suggestion
    cp packages/app-lib/.env.prod packages/app-lib/.env
```

This should be `env.prod` of course for the launcher to load content from modrinth's website :D (fixes the "Unable to read loader tags from any source" error)

---

Ooh thanks :)

I did basically zero research on this and just guessed: I was thinking of `local` like local machine, that will run for the end user and `prod` as a production *server* (since it's a monorepo with all their code?). But `local` is more like `dev` in most projects.